src/components/CallAssignmentList.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import { Flex, Text, View } from '@adobe/react-spectrum';
+
+import { ZetkinCallAssignment, ZetkinMembership } from '../types/zetkin';
+
+interface CallAssignmentListProps {
+    callAssignments: ZetkinCallAssignment[] | undefined;
+    memberships: ZetkinMembership[] | undefined;
+}
+
+export default function CallAssignmentList ({ callAssignments, memberships } : CallAssignmentListProps) : JSX.Element {
+
+    if (!callAssignments || callAssignments.length === 0) {
+        return (
+            <Text data-testid="no-assignments-placeholder">
+                <Msg id="misc.callAssignmentList.placeholder"/>
+            </Text>
+        );
+    }
+
+    return (
+        <>
+            { callAssignments.map((call) => {
```

Comment:
It can become confusing if assigments are stored in a variable called `call`, since calls are something else. In a call assignment, you can make actual phone calls, so you will often see both call assignments and calls appearing in the same contexts in code.